proj/src/main/webapp/signup.js

```diff
@@ -74,7 +75,8 @@ async function signUp(fname, lname, email, pass, tagStr, lat, long){
                 tagRemovalDict : tagRemovalDict,
                 latitude : lat,
                 longitude : long,
-                bio : "I'm a new user! Say hi!"
+                bio : "I'm a new user! Say hi!",
+                photo: defaultPfp,
```

Comment:
I think it would be better to export the PFP in chat and use it from chat as `photo: chat.DEFAULT_PFP`. to export it from chat you just need to add `exports.DEFAULT_PFP = DEFAULT_PFP` to the bottom of chat.js